trySwift/SessionsTableViewController.swift

```diff
@@ -108,33 +109,33 @@ extension SessionsTableViewController: IndicatorInfoProvider {
 }
 
 extension SessionsTableViewController: UIViewControllerPreviewingDelegate {
-    
+
     func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
         guard let indexPath = tableView.indexPathForRow(at: location) else { return nil }
         // This will show the cell clearly and blur the rest of the screen for our peek.
         previewingContext.sourceRect = tableView.rectForRow(at: indexPath)
         let session = conferenceDay.sessionBlocks[indexPath.section].sessions[indexPath.row]
         return viewController(for: session)
     }
-    
+
     func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
         scheduleViewController?.performSegue(withIdentifier: sessionDetailsSegue, sender: viewControllerToCommit)
     }
 }
 
 extension SessionsTableViewController {
-    
+
     func configureTableView() {
-        
+
         tableView.register(SessionTableViewCell.self)
-        
+
         tableView.estimatedRowHeight = 160
         tableView.rowHeight = UITableViewAutomaticDimension
     }
 }
 
 private extension SessionsTableViewController {
-    
```

Comment:
not to brag but phabricator has a feature of hiding whitespace changes in review - it's pretty cool because it minimizes noise like this in review. I remember there was some sort of secret url parameter when reviewing pull requests… but to be honest I don't even know when I would want to see whitespace changes 🙂

Reply:
https://blog.github.com/2011-10-21-github-secrets/ 😀